### PR TITLE
[Filter] reduce duplicated source

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, add common functions for combi option and set-info callback.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -2113,6 +2113,96 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
...
+          goto error;
+        }
+
+        gst_tensor_info_copy (&combined->info[idx++], &out->info[i]);
+      }
+    }
+
+    combined->num_tensors = idx;
+  } else {
+    gst_tensors_info_copy (combined, out);
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:2162]: A tensor name of combined info is a resource leak.
```suggestion
    gst_tensors_info_free (combined, out);
```

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 2]
```diff
@@ -2113,6 +2113,96 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
...
+  }
+
+  return TRUE;
+
+error:
+  gst_tensors_info_init (combined);
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:2168]: A tensor name of the combined info is resource leak.
```suggestion
  gst_tensors_info_free (combined);
```

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c]: updated

**Review by gichan-jang — APPROVED** [review]
LGTM (:

**Comment by myungjoo** [comment]
Great Approach!

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 3]
```diff
@@ -2113,6 +2113,96 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
...
+  g_return_val_if_fail (in != NULL, FALSE);
+  g_return_val_if_fail (out != NULL, FALSE);
+  g_return_val_if_fail (combined != NULL, FALSE);
+
+  gst_tensors_info_init (combined);
+
+  if (priv->combi.out_combi_i_defined || priv->combi.out_combi_o_defined) {
+    if (priv->combi.out_combi_i_defined) {
+      for (list = priv->combi.out_combi_i; list != NULL; list = list->next) {
+        i = GPOINTER_TO_INT (list->data);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:2136]: Better to use `GPOINTER_TO_UINT`?

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**